### PR TITLE
test(ui): document creation tests

### PR DESCRIPTION
**Pull request by yadavyk**
An automated Cypress test to create and validate attributes of:

1. Item
2. Lead
3. Supplier

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://test_site:8000/",
+  "baseUrl": "http://site:8000/",
```
Comment (ankush) [inline cypress.json]: why is this being changed?

Comment (yadavyk) [reply cypress.json]: Was trying to change it on my local repo, guess I messed it up here

Comment (ankush) [inline cypress.json]:
```suggestion
  "baseUrl": "http://test_site:8000/",
```

File: cypress/integration/test_item.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		cy.findByRole('button', {name: /Edit in full page/}).click();
+        cy.get_field('item_code', 'Data').type("ITM-0018");
+        cy.get_field('item_group', 'Link').type('All Item Groups');
+		cy.get_field('valuation_rate', 'Data').clear().type('8000');
+		cy.get_field('stock_uom', 'Link').clear().type('Nos');
+        cy.wait(500);
+		cy.findByRole('button', {name: 'Save'}).click();
+    });
+
+    it('Check Item attributes', () => {
```
Comment (ankush) [inline cypress/integration/test_item.js]:
```suggestion


```

Dependent test. Should be merged ideally.

File: cypress/integration/test_lead.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+		cy.get_field('last_name', 'Link').type("Eleanor");
+		cy.get_field('gender', 'Link').type("Male");
+		cy.get_field('email_id', 'Link').type("oliverel@gmail.com");
+		cy.findByRole('button', {name: 'Save'}).click();
+		cy.get_field('salutation', 'Link').should('have.value', 'Mr');
+		cy.get_field('company_name', 'Link').should('have.value', 'Eleanor School of Music');
+		cy.get_field('first_name', 'Link').should('have.value', 'Oliver');
+		cy.get_field('last_name', 'Link').should('have.value', 'Eleanor');
+		cy.get_field('email_id', 'Link').should('have.value', 'oliverel@gmail.com');
+        	cy.location("pathname").should("not.be","/app/lead/new");
```
Comment (ankush) [inline cypress/integration/test_lead.js]:
```suggestion
        cy.location("pathname").should("not.be","/app/lead/new");
```

File: cypress/integration/test_lead.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+		cy.remove_doc('Lead', 'CRM-LEAD-2022-00003');
```
Comment (ankush) [inline cypress/integration/test_lead.js]:
```suggestion
```

This is failing in CI. Your local document naming wont be there in test env, so you need to somehow acquire name from the doc you created or just leave it in there.

**Review by ankush — DISMISSED** [review]

**Comment by ankush** [comment]
Screenshot 2022-03-24 at 11 28 07 AM



supplier test failing because of quick entry window.

**Comment by yadavyk** [comment]
> Screenshot 2022-03-24 at 11 28 07 AM
> 
> supplier test failing because of quick entry window.

This should work now

**Pull request merged**